lib/fetch.go

```diff
@@ -203,6 +222,11 @@ func (of *OCIFetcher) fetchManifest(u *URL) (*schema.ImageManifest, error) {
 		return nil, fmt.Errorf("unexpected http code: %d, URL: %s", res.StatusCode, req.URL)
 	}
 
+	digest := res.Header.Get("Docker-Content-Digest")
```

Comment:
I'm pretty sure sha256sum will match (since I'm doing the same in the oci store WriteBlob function)